gateway-service-knoxsso/pom.xml

```diff
@@ -81,5 +81,9 @@
             <artifactId>gateway-test-utils</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.knox</groupId>
+            <artifactId>gateway-server</artifactId>
```

Comment:
Which class is references from gateway-server from gateway-service-knoxssou?

Reply:
The ConcurrentSessionVerifier is in gateway-server.

Reply:
Same as about. This can be deleted by inverting the dependencies.

Reply:
Thats a good idea thank you, i implemented it.